getting_started.md

```diff
@@ -0,0 +1,169 @@
+# Getting started
+
+This is a short tutorial that runs you through some basic functionality of the poseidon framework and tooling.
+
+## Preparation
+You will need to install our two command-line tools `trident` and `xerxes`. They are available as pre-compiled binaries for MacOS and Linux, and in case of trident also for Windows: 
+* `trident`: https://github.com/poseidon-framework/poseidon-hs/releases/latest
+* `xerxes`: https://github.com/poseidon-framework/poseidon-analysis-hs/releases/latest
+
+You can also install both tools from bioconda: [trident](https://anaconda.org/bioconda/poseidon-trident), [xerxes](https://anaconda.org/bioconda/poseidon-xerxes)
```

Comment:
The versions in (on?) bioconda are probably outdated, right? Are our releases there stable enough to recommend them for beginners?

Reply:
I have updated them right now... and if we advertise the conda install in the READMEs, I think they should appear here too, no?

Reply:
Alright